Src/Support/GoogleApis.Core/Apis/Http/ConfigurableMessageHandler.cs

```diff
@@ -173,6 +174,19 @@ public void RemoveExecuteInterceptor(IHttpExecuteInterceptor interceptor)
 
         #endregion
 
+        private int _loggingRequestId = 0;
+
+        private ILogger _instanceLogger = Logger;
+
+        /// <summary>
+        /// For testing only

```

Comment:
Unfortunately we've ended up with different styles on different projects. Certainly for the google-cloud-dotnet and gax-dotnet projects we're using underscores (not my preference, but I can live with it). I can't remember what the general style is in this repo - a very quick look suggests it's to *not* have leading undescores...